test/test_utils.py

```diff
@@ -79,6 +79,15 @@ def test_save_image_single_pixel_file_object(self):
             self.assertTrue(torch.equal(F.to_tensor(img_orig), F.to_tensor(img_bytes)),
                             'Pixel Image not stored in file object')
 
+    def test_draw_boxes(self):
+        img = torch.randint(0, 255, (3, 226, 226), dtype=torch.uint8)
+        boxes = torch.tensor([[0, 0, 100, 100], [0, 0, 0, 0],
+                             [10, 15, 30, 35], [23, 35, 93, 95]], dtype=torch.float)
+        labels = ['a', 'b', 'c', 'd']
+        utils.draw_bounding_boxes(img, boxes, labels=labels)
+
+        return True
```

Comment:
Do we need this return?

Reply:
No we don't but the test code was not done yet. 

At the moment we are not asserting anything other than the fact that no exception is thrown. One way is to produce the result using the method and then store it in the assets and assert against that. It's not the best but it would be better than nothing. Thoughts?

Reply:
Marking as resolved because the return was removed. Let's discuss the test details on the other thread.